### ISSUE TITLE
[Guidance] Power Platform Landing Zones for brownfield

### ISSUE THREAD

**Issue by krnese** [microsoft/industry#312]

**Comment by aydinderya** [comment]
@krnese @daltondhcp thank you for the update, as this item has the backlog label, could you please assign a team member? Thank you

**Comment by krnese** [comment]
Yes @aydinderya - we're working to figure out who to assign this effort the coming days.
